### ISSUE TITLE
[macOS] Can't access Bento menu using keyboard Navigation

### ISSUE THREAD

**Issue by sv-hyacoub** [mozilla/blurts-server#1217]
[Affected versions]:
Firefox 69.0.1, Safari

[Affected Platforms]:
Firefox 69.0.1 only on macOS 10.13
Safari on all MacOS's

[Steps to reproduce]:
1. Navigate to the Monitor website "https://monitor-v2.herokuapp.com/".
2. Using "Tab" key, open Bento menu.
3. Using "Tab" key, access Bento menu options.

[Expected result]:
Bento menu options should be accessed by keyboard only.

[Actual result]:
Can't access Bento menu options using keyboard Navigation.

[Note]:
- The issue on Firefox 69.0.1 is only happening on macOS 10.13
- The issue on Safari in happening on all MacOS versions.

**Comment by lesleyjanenorton** [comment]
I can't reproduce in either browser. Are your system preferences configured for key command navigation? These can particularly affect Safari. This might be helpful https://a11yproject.com/posts/safari-keyboard-navigation/.

**Comment by lesleyjanenorton** [comment]
Closing as fixed - please reopen if you're able to reproduce this.

**Comment by sv-hyacoub** [comment]
Verified as fixed on Firefox 69.0.1 and Safari on macOS10.13 and 10.14.
